app/Http/Controllers/HomePage/HomeController.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
I think you don't need this file

Reply:
I think it's necessary, how can customers order if they can't see the product